src/styles/fonts.css

```diff
@@ -0,0 +1,11 @@
+@font-face {
+  font-family: Avenir;
+  src: url('../fonts/Avenir-Book.ttf');
+  font-weight: 300;
```

Comment:
The `font-weight` for Avenir-Book is set to 300, which is typically associated with 'Light' font weights. If Avenir-Book is meant to represent a 'Regular' weight, you should use 400 instead.